src/app/projects/[project_id]/page.tsx

```diff
@@ -163,6 +164,16 @@ export default async function Project({
 
   return (
     <Container maxWidth="lg" className="project-box">
+      <Button
+        startIcon={
+          <ArrowBackIosIcon className="project-box__back-button-icon" />
+        }
+        className="project-box__back-button"
+        href="./project"
```

Comment:
```suggestion
        href="/projects"
```

https://github.com/user-attachments/assets/a44138aa-e78f-4829-b5d3-1ee11bed095b

Seems the deployment on Vercel breaks if the `href` is set to `./project`

I'm recommending the `href` be set to `/projects`, given what I set the `redirect` to here before:

https://github.com/CodeForBc/codeforbc-web/blob/24a1b019c3570a2cd2d6b9cdec328aff44d7c468/src/app/projects/%5Bproject_id%5D/page.tsx#L160-L162